CityOfDetroit/bloom
# Add filtering for rent price

## Issue

- Closes #102 

## Description

Adds ability to filter by rent price based on data in the units summary table.

UI:
![image](https://user-images.githubusercontent.com/83078310/132047846-fa7495a0-5678-4b98-a160-9864b869edb3.png)


## Type of change


## How Can This Be Tested/Reviewed?

Performed some filtering a verified that the query was hitting the backend and the data was being filtered based on the units summary table.

NOTE: this is a little broken in certain cases, e.g. if you want to search for 2-bedrooms that are less than $600 a month, because the bedroom filter is looking at the units table, and the rent filter is looking at the summary table. We need way to generally determine which tables we should be joining to make queries. That's tracked in #530.


## Checklist:
